handlers.py

```diff
@@ -72,6 +72,27 @@ def get_all_responses_with_game(self, message, game):
             return []
 
 
+class SameHandler(GameExtractionMixin, ContentBasedHandler):
+    fragments = ['same', 'Same']
+
+    def get_all_responses_with_game(self, message, game):
+        would_play = []
+        fragment_present = any(message.content.lower().startswith(f.lower()) for f in self.fragments)
```

Comment:
I think the fragment has to be present for this to run (from should_handle_message)